test/extensions/KHR/SPV_KHR_cooperative_matrix/matrix_times_scalar.spt

```diff
@@ -0,0 +1,44 @@
+; RUN: llvm-spirv %s -to-binary -o %t.spv
+; Validation is disabled till the moment the tools in CI are updated
```

Comment:
It may help to add "TODO" here, to increase the chances of catching it with a `grep` in the future.

```suggestion
; TODO: Validation is disabled till the moment the tools in CI are updated
```